.circleci/config.yml

```diff
@@ -185,19 +185,12 @@ jobs:
       - run:
           name: release beta homebrew
           command: |
-            cd /tmp
-
-            wget https://artsy-provisioning-public.s3.amazonaws.com/hokusai/hokusai-beta-Darwin-arm64.tar.gz
-
-            export UNAME=$(uname)
-            if [ $UNAME == "Darwin" ]; then
-              export SHA256=$(shasum -a 256 hokusai-beta-Darwin-x86_64.tar.gz | awk '{ print $1 }')
-            fi
+            export ARTIFACT=hokusai-beta-Darwin-arm64.tar.gz
 
-            if [ $UNAME == "Linux" ]; then
-              export SHA256=$(sha256sum hokusai-beta-Darwin-x86_64.tar.gz | awk '{ print $1 }')
-            fi
```

Comment:
i don't see the use of these `if`s. the job runs in docker container, so `uname` is always `Linux`.